tests/ui/pages/content.py

```diff
@@ -0,0 +1,50 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from .base import Page
+from ..regions.base import Region
+
+
+class Content(Page):
+    URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
+
+    _main_content_locator = (
+        By.CSS_SELECTOR,
+        '#main-content > div > div > div h1[data-type="document-title"]',
+    )
+    _table_of_contents_button_locator = (By.CSS_SELECTOR, "#root > div > div > button")
+    _table_of_contents_nav_locator = (By.CSS_SELECTOR, "#root > div > div > div > div > nav")
```

Comment:
@philschatz should weigh in but i think we should just sprinkle some `aria-label`s on these to select them by

Reply:
I agree. They didn't exist so this is what i did to get the tests to identify the right elements.